src/Type/Comment/CommentType.php

```diff
@@ -123,7 +123,14 @@ private static function fields() {
 						'type'        => Types::string(),
 						'description' => __( 'Content of the comment. This field is equivalent to WP_Comment->comment_content and the value matching the `comment_content` column in SQL.', 'wp-graphql' ),
 						'resolve'     => function( \WP_Comment $comment, $args, AppContext $context, ResolveInfo $info ) {
-							return ! empty( $comment->comment_content ) ? $comment->comment_content : '';
+							$content =  ! empty( $comment->comment_content ) ? $comment->comment_content : null;
+
+							// If the raw format is requested, don't apply any filters.
+							if ( isset( $args['format'] ) && 'raw' === $args['format'] ) {
+								return $content;
+							}
+
+							return apply_filters('comment_text', $content);
```

Comment:
@kidunot89 good addition! We need to also support the `format` argument for this field then

Reply:
I already add support for that. It's there as well

Reply:
@kidunot89 where? I don't see it in this diff unless I'm overlooking something 🤔 👀 

I would expect to see it here with the field definition: 

```
  'type'        => Types::string(),
  'description' => __( 'Content of the comment. This field is equivalent to WP_Comment->comment_content and the value matching the `comment_content` column in SQL.', 'wp-graphql' ),
'args' => [
  'format' => [
     'type' => // the format enum
     'description' => // the description of the enum 
  ]
]
```

Reply:
Corrected. I used Types::post_object_field_format_enum() as the type